### PR TITLE
support workload operator

### PR DESCRIPTION
**Pull request by harshad16**
Add missing image stream and build config of workload operator and Amun-API.
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -82,11 +82,13 @@
     - https://raw.githubusercontent.com/thoth-station/result-api/master/openshift/buildConfig-template.yaml
     - https://raw.githubusercontent.com/thoth-station/solver/master/openshift/buildConfig-template.yaml
     - https://raw.githubusercontent.com/thoth-station/user-api/master/openshift/buildConfig-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/workload-operator/master/openshift/imageStream-template.yaml
 
 - name: create Thoth Core ImageStream templates
   command: oc apply --namespace "{{ namespace  }}" --filename "{{ item }}"
   with_items:
     - https://raw.githubusercontent.com/thoth-station/adviser/master/openshift/imageStream-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/amun-api/master/openshift/imagestream-template.yaml
```
Comment (bissenbay) [inline tasks/main.yml]: Is line 90 and 91 the same thing?

Comment (bissenbay) [inline tasks/main.yml:91]: ImageStream?

Comment (harshad16) [reply tasks/main.yml:91]: The imagestream-template is already merged in amun-api, may be next time we can change the name.

**Review by bissenbay — APPROVED** [review]
Thanks, :1st_place_medal: !

**Pull request merged**